### ISSUE TITLE
wrong svg parse

### ISSUE THREAD

**Issue by samuelgjabel** [wcandillon/react-native-redash#473]
I have mismatch with parsing svgPath...

if I parse('M 0 12.5 L 28 12.5 M 12.5 0 L 12.5 28')

it returns serialized:  M 0 12.5 C 0 12.5 28 12.5 28 12.5 C 12.5 0 12.5 28 12.5 28
which is wrong svg.

Currently I looked at the code & if i do ` const segments: SVGNormalizedCommands = normalizeSVG(absSVG(parseSVG(d)))`
& serialize segments back to the string, it work ok. So problem is somewhere else :( 

Thanks for help! :)

**Comment by wcandillon** [comment]
Indeed, it normalizes path as a sequence of bezier curves (for interpolation) so it won't return the same commands but the path should look the same when you draw it. does this helps?
